airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/destination_async/state/PartialStateWithDestinationStats.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.integrations.destination_async.state;
+
+import io.airbyte.cdk.integrations.destination_async.partial_messages.PartialAirbyteMessage;
+import io.airbyte.protocol.models.v0.AirbyteStateStats;
```

Comment:
I would have expected to see a version bump of the protocol package to get this new `AirbyteStateStats` object.

Reply:
Well turns out you dont! haha! The versioning thing is different. v0 is still in prod. We need to put in work to migrate everything to v1.